doc/conf.py

```diff
@@ -139,7 +139,8 @@
                   ('py:class', 'glue.viewers.image.layer_artist.RGBImageLayerBase'),
                   ('py:class', 'glue.viewers.image.state.BaseImageLayerState'),
                   ('py:class', 'sip.voidptr'),
-                  ('py:class', 'PyQt5.sip.voidptr')]
+                  ('py:class', 'PyQt5.sip.voidptr'),
+                  ('py:class', 'PYQT_SLOT')]
```

Comment:
```suggestion
                  ('py:class', 'QKeySequence.StandardKey')],
                  ('py:class', 'PYQT_SLOT')]
```
Another one, in `BasicToolbar.grabShortcut`?